### PR TITLE
Docs: safe-buffer usage note, Node.js distribution table

### PR DESCRIPTION
**Pull request by holgerd77**

### CODE REVIEW

File: docs/technical.rst [hunk 1]
```diff
@@ -21,17 +21,28 @@ Runtime environment for development is `node.js <https://nodejs.org/en/>`_.
...
 see Node.js `release schedule table <https://github.com/nodejs/Release#release-schedule>`_.
 
-====================== =================================================
-Node.js 8              Supported
-Node.js 10             Supported
-====================== =================================================
+====================== ================= ===============================
+Node Version           Status            Latest Status Change
+====================== ================= ===============================
+Node.js 8              Supported         2018-11-01
```
Comment (axic) [inline docs/technical.rst]: How about version 6?

File: docs/technical.rst [hunk 2]
```diff
@@ -162,6 +173,21 @@ compiler ``tsc`` command line tool.
...
+
+The following table gives an overview on supported Node versions for distribution:
+
+====================== ================= ===============================
+Node Version           Status            Latest Status Change
+====================== ================= ===============================
+Node.js 4              Dropped           2018-10-01
+Node.js 6              Supported         2018-10-01
+Node.js 8              Supported         2018-11-01
+Node.js 10             In the works      2018-12-01
```
Comment (whymarrh) [inline docs/technical.rst]: I think its safe to call Node.js 10 supported here as all existing code should run on it.

File: docs/technical.rst [hunk 3]
```diff
@@ -21,17 +21,28 @@ Runtime environment for development is `node.js <https://nodejs.org/en/>`_.
...
+Buffer vs safe-buffer
+"""""""""""""""""""""
+Old Node versions up to versions ``4`` and ``5`` allowed for some unsafe usage
+of the ``Buffer`` API which led to the development of a replacement 
+`safe-buffer <https://github.com/feross/safe-buffer>`_ library. We have now for
+some time dropped support for distibution on all affected Node versions (mainly
+``4``) and usage of ``safe-buffer`` is not needed any more.
 
-Notes on ``Buffer``, ``safe-buffer``, stuff like that.
+.. note::
```
Comment (whymarrh) [inline docs/technical.rst]: I think this note might not be needed. I can instead remove all of the `safe-buffer` usages.

File: docs/technical.rst [hunk 4]
```diff
@@ -21,17 +21,28 @@ Runtime environment for development is `node.js <https://nodejs.org/en/>`_.
...
+====================== ================= ===============================
 
 Node.js Features
 ^^^^^^^^^^^^^^^^
 
-``TODO``
+Buffer vs safe-buffer
+"""""""""""""""""""""
+Old Node versions up to versions ``4`` and ``5`` allowed for some unsafe usage
+of the ``Buffer`` API which led to the development of a replacement
```
Comment (axic) [inline docs/technical.rst]: It isn't really a replacement, but a polyfill. Maybe it was first, it is just nitpicking :)

File: docs/technical.rst (hunk 2 above)
Comment (holgerd77) [reply docs/technical.rst]: We had some problems with Node.js 10 lately - mainly due to dependencies not being up-to-date to play well with Node.js 10, e.g. https://github.com/ethereumjs/ethereumjs-client/issues/53 - so I would like to keep this on this "in the works" state a bit until we have got a bit more assurance that most code/dependencies are in line.

File: docs/technical.rst (hunk 3 above)
Comment (holgerd77) [reply docs/technical.rst]: Will keep this until `safe-buffer` usage is removed.

**Pull request merged**